lib/views/physician/physician_profile.dart

```diff
@@ -0,0 +1,85 @@
+import 'package:age_calculator/age_calculator.dart';
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:flutter/material.dart';
+import 'package:pallinet/components/patient_card.dart';
+import 'package:pallinet/constants.dart';
+import 'package:pallinet/firestore/firestore.dart';
+import 'package:pallinet/models/patient_model.dart';
+
+class PhysicianProfile extends StatelessWidget {
+  const PhysicianProfile({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(title: const Text('Edit Physican Profile')),
+        body: const Padding(padding: EdgeInsets.all(16.0), child: ProfileContent()));
+  }
+}
+
+class ProfileContent extends StatefulWidget {
+  const ProfileContent({Key? key}) : super(key: key);
+
+  @override
+  State<ProfileContent> createState() => ProfileContentState();
+}
+
+class ProfileContentState extends State<ProfileContent> {
+  bool isPasswordVisible = false;
+
+  final GlobalKey<FormState> _formKey = GlobalKey<FormState>();
+
+  PatientID? patient;
+  List? practitioners = [];
+  DateTime? date = DateTime.now();
+  String? desc = "";
+  ServiceType? serviceType;
+
+  @override
+  Widget build(BuildContext context) {
+    return FutureBuilder<List<PatientID>>(
+      future: retrievePatients2(),
```

Comment:
Retrieve patients here doesn't really make sense, though I'm assuming it's just a placeholder for now.

Reply:
It's probably fine for now, but make sure in your next task where you're connecting to the database, you remove the code that isn't related.